test/integration/kubernetes_deploy_test.rb

```diff
@@ -806,6 +806,49 @@ def test_invalid_context
     end
   end
 
+  def test_multiple_configuration_files
+    old_config = ENV['KUBECONFIG']
+    begin
```

Comment:
you actually don't need to `begin...end` here if you outdent the `ensure` to match `def`; the body of a ruby method definition is an implicit `begin...end` block.

Reply:
Another trick to learn :-)